File: Commons/Math/src/main/java/com/greentree/commons/math/vector/ord/Value.java
package com.greentree.commons.math.vector.ord;


public class Value implements ChangeableFloatVectorOrd {

	private float value;
	
	public Value() {
	}
	
	public Value(float value) {
		this.value = value;
	}

	@Override
	public float get() {
		return value;
	}

	@Override
	public void set(float x) {
		value = x;
	}

}

File: Commons/Math/src/main/kotlin/com/greentree/commons/math/vector/ord/Value.kt
package com.greentree.commons.math.vector.ord

class Value : ChangeableFloatVectorOrd {
    private var value = 0f

    constructor()
    constructor(value: Float) {
        this.value = value
    }

    override fun get(): Float {
        return value
    }

    override fun set(x: Float) {
        value = x
    }
}